### PR TITLE
feat : added single video page

### PR DESCRIPTION
**Pull request by Sharmaryan**
added single video page

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -9,7 +9,8 @@ import {
   WatchLater,
   Explore,
   LoginPage,
-  SignupPage
+  SignupPage,
+  VideoDetail
 } from "./pages/index";
```
Comment (ravikumar0403) [inline src/App.js:14]: `/index` is not required here

File: src/pages/VideoDetail/VideoDetail.js [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+export const VideoDetail = () => {
+  const { videoId } = useParams();
+  const [singleVideoDetail, setSingleVideoDetail] = useState(null);
+
+  useEffect(() => {
+    (async () => {
+      try {
+        const response = await axios.get(`/api/video/${videoId}`);
+        console.log(response.data.video);
```
Comment (ravikumar0403) [inline src/pages/VideoDetail/VideoDetail.js]: remove console logs

**Review by ravikumar0403 — APPROVED** [review]
Liked the usage of nullish coalescing operator
Code looks clean, I've left a couple of sanity comments

**Review by saketkum11 — APPROVED** [review]
- good implement of code 
- file structure is good 
- easy to understand 
-

**Pull request merged**